app/views/application/edit-response.njk

```diff
@@ -31,6 +31,7 @@
     </div>
   </div>
 
+<p class="govuk-body">This candidate has not accepted your offer yet. This means you can withdraw it, or offer a different course.</p>
```

Comment:
I don't think we should add this line yet. 

I'd like to find out how it goes in research without it so that we can work out what the solution is as it may not be to add guidance and explain how things work.

Reply:
So far, with just 1 participant there was no issue. will continue to keep an eye.